src/oemof/tabular/facades.py

```diff
@@ -1578,7 +1578,33 @@ def __init__(self, *args, **kwargs):
 
         self.marginal_cost = kwargs.get("marginal_cost")
 
-        self.inputs.update({self.bus: Flow(variable_costs=self.marginal_cost)})
+        self.capacity = kwargs.get("capacity")
+
+        self.capacity_potential = kwargs.get(
+            "capacity_potential", float("+inf")
+        )
+
+        self.capacity_cost = kwargs.get("capacity_cost")
+
+        self.capacity_minimum = kwargs.get("capacity_minimum")
+
+        self.expandable = bool(kwargs.get("expandable", False))
+
+        self.input_parameters = kwargs.get("input_parameters", {})
+
+        self.build_solph_components()
+
+    def build_solph_components(self):
+        """
+        """
+        f = Flow(
+            nominal_value=self._nominal_value(),
+            variable_costs=self.marginal_cost,
+            investment=self._investment(),
+            **self.input_parameters
+        )
```

Comment:
Could it be somehow of interest to allowe a profile for Excess as it is for Shortage?

```suggestion
        self.profile = kwargs.get("profile", 1)

        self.capacity = kwargs.get("capacity")

        self.capacity_potential = kwargs.get(
            "capacity_potential", float("+inf")
        )

        self.capacity_cost = kwargs.get("capacity_cost")

        self.capacity_minimum = kwargs.get("capacity_minimum")

        self.expandable = bool(kwargs.get("expandable", False))

        self.input_parameters = kwargs.get("input_parameters", {})

        self.build_solph_components()

    def build_solph_components(self):
        """
        """
        if self.profile is None:
            self.profile = 1
        
        f = Flow(
            nominal_value=self._nominal_value(),
            variable_costs=self.marginal_cost,
            max=self.profile,
            investment=self._investment(),
            **self.input_parameters
        )
```

Reply:
Could be, in the future. I would leave it out for the moment if there is no urgent reason. I will make a note in the issue.